pkg/config/log.go

```diff
@@ -55,25 +72,31 @@ func SetupLogger(logLevel, logFile, uri string, rfc, tls bool, pem string, logTo
 				`<custom name="syslog" formatid="syslog" data-uri="%s" data-tls="%v" />`,
 				uri,
 				tls,
-			)
+				)
 		} else {
 			syslogTemplate = `<custom name="syslog" formatid="syslog" />`
 		}
 		configTemplate += syslogTemplate
 	}
+
 	configTemplate += `</outputs>
-    <formats>
-        <format id="common" format="%%Date(%s) | %%LEVEL | (%%File:%%Line in %%FuncShort) | %%Msg%%n"/>`
+	<formats>`
+
+	if jsonFormat {
+		configTemplate += jsonOutput
+	} else {
+		configTemplate += commonOutput
+	}
 	if syslog {
-		if rfc {
-			configTemplate += `<format id="syslog" format="%%CustomSyslogHeader(20,true) %%LEVEL | (%%RelFile:%%Line) | %%Msg%%n" />`
+		if jsonFormat {
+			configTemplate += `<format id="std:json" format="{&quot;header&quot;:&quot;%%CustomSyslogHeader(20,`+ strconv.FormatBool(rfc) +`)&quot;,&quot;level&quot;:&quot;%%LEVEL&quot;,&quot;relfile&quot;:&quot;%%RelFile&quot;,&quot;line&quot;:&quot;%%Line&quot;,&quot;msg&quot;:&quot;%%Msg&quot;}%%n"/>`
```

Comment:
@hush-hush , I reduced the if-else statements here by directly putting the boolean value of `rfc` in the format tag itself. Is there a better way you might suggest to clean up this function?